src/com/itschool/session4/homework/challenge03/LogicalOperators.java

```diff
@@ -0,0 +1,25 @@
+package com.itschool.session4.homework.challenge03;
+
+import java.util.Scanner;
+public class LogicalOperators {
+
+    public static void main(String[] args) {
+        //creaza un nou obiect  Scanner
+        Scanner obiecScaner = new Scanner(System.in);
```

Comment:
typo here. I think you meant `obiectScanner`.

Reply:
thx for observations